tests/main/protocol-spec.ts

```diff
@@ -41,6 +41,20 @@ describe('protocol', () => {
   });
 
   describe('listenForProtocolHandler()', () => {
+    it('handles a Fiddle url (second-instance)', () => {
+      overridePlatform('win32');
+
+      listenForProtocolHandler();
+
+      const handler = (app.on as any).mock.calls[1][1];
```

Comment:
nit: can we assert somewhere that `app.on` was also called with the `second-instance` event parameter?

Reply:
```suggestion
      const handler = (app.on as any).mock.calls[1][1];
```